BibBuddy/app/src/main/java/de/bibbuddy/NoteRecyclerViewAdapter.java

```diff
@@ -67,81 +59,61 @@ public MyViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     Context context = parent.getContext();
     itemView =
         LayoutInflater.from(context).inflate(R.layout.list_view_item_note, parent, false);
-    background = itemView.getBackground();
     hiddenDeletePanel = parent.getRootView().findViewById(R.id.hidden_delete_panel);
     panelDelete = hiddenDeletePanel.findViewById(R.id.panel_delete);
     return new MyViewHolder(itemView);
   }
 
   /**
-   * Method to setup the custom ViewHolder components
-   * Consider: if text has more than 40 characters, only first 35 characters are displayed as title
-   * and adding " ..." as indicator for a longer text
+   * Method to setup the custom ViewHolder components for notes.
    *
    * @param holder   custom ViewHolder instance
    * @param position adapterPosition of the viewHolder-item
    */
   @Override
   public void onBindViewHolder(@NonNull MyViewHolder holder, int position) {
     Long id = data.get(position).getId();
+    String text = data.get(position).getNoteText();
     setupCardView(holder, position);
-    holder.itemView.setOnClickListener(new View.OnClickListener() {
-      @Override
-      public void onClick(View v) {
-        TextNoteEditorFragment nextFrag = new TextNoteEditorFragment();
-        Bundle args = new Bundle();
-        args.putLong(LibraryKeys.NOTE_ID, id);
-        nextFrag.setArguments(args);
-        activity.getSupportFragmentManager().beginTransaction()
-            .replace(R.id.fragment_container_view, nextFrag, "fragment_text_note_editor")
-            .addToBackStack(null)
-            .commit();
-      }
+    holder.itemView.setOnClickListener(v -> {
+      TextNoteEditorFragment nextFrag = new TextNoteEditorFragment();
+      Bundle args = new Bundle();
+      args.putLong(LibraryKeys.NOTE_ID, id);
+      args.putString(LibraryKeys.NOTE_TEXT, text);
+      nextFrag.setArguments(args);
+      activity.getSupportFragmentManager().beginTransaction()
+          .replace(R.id.fragment_container_view, nextFrag,  LibraryKeys.FRAGMENT_TEXT_NOTE_EDITOR)
+          .addToBackStack(null)
+          .commit();
     });
-    holder.itemView.setOnLongClickListener(new View.OnLongClickListener() {
-
-      @Override
-      public boolean onLongClick(View v) {
-
-        // Below line is just like a safety check, because sometimes holder could be null,
-        // in that case, getAdapterPosition() will return RecyclerView.NO_POSITION
-        if (position == RecyclerView.NO_POSITION) {
-          return false;
-        }
-        if (!v.isSelected()) {
-          v.setSelected(true);
-          v.setBackgroundColor(activity.getColor(R.color.flirt_light));
-        } else {
-          v.setSelected(false);
-          v.setBackground(background);
-        }
-        slideUpOrDown();
-        return true;
+    holder.itemView.setOnLongClickListener(v -> {
```

Comment:
Newline for better readability.
```suggestion

    holder.itemView.setOnLongClickListener(v -> {
```